src/passkit.cs

```diff
@@ -984,8 +984,33 @@ interface PKLabeledValue
 		string Value { get; }
 	}
 
+	[Watch (4,3), iOS (11,3)]
+	[BaseType (typeof (NSObject))]
+	interface PKTransitPassProperties {
+
+		[Static]
+		[Export ("passPropertiesForPass:")]
+		[return: NullAllowed]
+		PKTransitPassProperties GetPassProperties (PKPass pass);
```

Comment:
that _hint_ to a `[DisableDefaultCtor]`, the fact that there's no setter seems to make it unusable if `init` was used (successfully).

looks like we should have done the same on `PKSuicaPassProperties` :|

Reply:
Ahh good catch!

Reply:
Ahh good catch! 👍